students/drovdahl/lesson08/circle.py

```diff
@@ -18,53 +18,49 @@
 
 class Circle(object):
     def __init__(self, radius):
-        self._radius = radius
+        self.radius = radius
         self.diameter = radius * 2
```

Comment:
You can also delete this "self.diameter = radius * 2" on line 22, because you defined diameter as a @property below and it already returns radius * 2.